prometheuscollector/prometheuscollector.go

```diff
@@ -46,11 +46,11 @@ var (
 )
 
 type Collector struct {
-	metrics tusd.Metrics
+	metrics *tusd.Metrics
```

Comment:
If you are talking about the one in the Collector struct, then I don't think so.